src/theme/colorThemes.js

```diff
@@ -3,20 +3,31 @@ const success = "#1bab1e";
 const global = {
   brand: "#026cdf",
   primary: {
+    // brand colors
     base: "#026cdf",
+    reverse: "#3396ff",
+    reverseLight: "#80b5ef",
+    reverseLightest: "#bfdaf6",
+
+    // button colors
```

Comment:
Design realizes these colors won't just be used for buttons, if we change the theme palette... right?

Reply:
I think they do. It's just what this group is called in the specs.

Reply:
If you think it's confusing I can delete this comment.

Reply:
It's fine to leave. Just want to be extra sure 😅